src/main/java/com/internet/shop/controllers/RegistrationController.java

```diff
@@ -34,6 +37,15 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp)
             User user = new User(name, login, psw);
             ShoppingCart shoppingCart = new ShoppingCart();
             user = userService.create(user);
+            Set<Role> roleSet = new HashSet<>();
+            if (user.getId() == 1) {
+                roleSet.add(Role.of("ADMIN"));
+                roleSet.add(Role.of("USER"));
+                user.setRoles(roleSet);
+            } else {
+                roleSet.add(Role.of("USER"));
+                user.setRoles(roleSet);
+            }
```

Comment:
let's set only USER role to everyone who registers

and inject Admin in InjectController (with only ADMIN role)

Reply:
fixed